docs/data-modeling/field-types.md

```diff
@@ -10,33 +10,35 @@ While data types indicate to a database how it should interpret the values in a
 
 Field types dictate how Metabase displays its data, as well as the column’s special functionality, if any. By marking columns in a table as **Latitude** and **Longitude**, Metabase can use the columns to create pin and heat maps. Similarly, designating a column as a **URL** allows users to click on the link to visit that URL.
 
-## List of Metabase Field Types
+## Overall Row
 
-Metabase recognizes the following field types:
+- **Entity Key**: The field in this table that uniquely identifies each row. Could be a product ID, serial number, etc.
+- **Entity Name**: Different from the entity key, the entity name represents what each row in the table _is_. For example, in a Users table, the User column might be the entity name.
+- **Foreign Key**: The column in this table that (usually) refers to the primary key of another table in order to connect data from different tables that are related. For example, in a Products table, you might have a Customer ID field that points to a Customers table, where Customer ID is the primary key.
 
-### Overall Row
+## Common
 
-- **Entity Key**. The field in this table that uniquely identifies each row. Could be a product ID, serial number, etc.
-- **Entity Name**. Different from the entity key, the entity name represents what each row in the table _is_. For example, in a Users table, the User column might be the entity name.
-- **Foreign Key**. The column in this table that (usually) refers to the primary key of another table in order to connect data from different tables that are related. For example, in a Products table, you might have a Customer ID field that points to a Customers table, where Customer ID is the primary key.
+Common field types are text/varchar types.
 
-### Common
-
-- **Category**
+- **Category**: If set to category, Metabase will know that it can use this field to group results when creating questions for automatic insights like [X-rays](../exploration-and-organization/x-rays.md).
 - **Comment**
 - **Description**
 - **Title**
 
-### Location
+## Location
+
+Marking fields as locations tell Metabase that it can use the field to display data on a [map chart](../questions/sharing/visualizations/map.md).
 
 - **City**
 - **Country**
-- **Latitude**
-- **Longitude**
+- **Latitude**: Tells Metabase that the data could be plotted on a [pin map](../questions/sharing/visualizations/map.md).
+- **Longitude**: Did you know that you can use the moons of Jupiter to calculate longitude?
 - **State**
 - **Zip Code**
 
-### Financial
+## Financial
+
+Financial field types tell Metabase to treat the field's values as money. If you select any of the financial field types, Metabase will ask which currency you want to use.
```

Comment:
So I'm a bit confused here, because this sentence makes me think that  Cost, Income etc have "currency" implicitly attached to it, sort of like timestamps can have timezones, but then there's also a currency semantic type? How do they interact? 

Also is there any difference in hoe Metabase handles, say, Cost vs Price?